src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -49,7 +50,7 @@
  * @author GroG
  * 
  */
-public class Python extends Service<PythonConfig> implements ServiceLifeCycleListener, MessageListener {
+public class Python extends Service<PythonConfig> implements ServiceLifeCycleListener, MessageListener, Processor {

```

Comment:
The exec method should have Override annotation

Reply:
updated